ui-showdetails/src/main/res/layout/view_holder_details_related_item.xml

```diff
@@ -60,10 +60,6 @@
             android:padding="@dimen/spacing_small"
             android:text="@{tiviShow.title}"
             android:textAppearance="?attr/textAppearanceBody2"
-            app:layout_constraintBottom_toBottomOf="parent"
```

Comment:
It's better to keep the explicit constraints and set the `layout_width` and `layout_height` to `match_constraint` or `0dp` (same thing)